alwaysontop/constants.js

```diff
@@ -21,7 +21,9 @@ module.exports = {
         HIDE: 'aot-hide',
         OPEN: 'aot-open',
         SHOW: 'aot-show',
```

Comment:
Quick (any maybe stupid) question: Whats the difference between the states "SHOW"/"HIDE" and "SHOW_AOT_WINDOW" / "HIDE_AOT_WINDOW"?

Reply:
> Quick (any maybe stupid) question: Whats the difference between the states "SHOW"/"HIDE" and "SHOW_AOT_WINDOW" / "HIDE_AOT_WINDOW"?

SHOW/HIDE are events sent from main process to notify renderer process that AOT was shown/hidden, while "SHOW_AOT_WINDOW" / "HIDE_AOT_WINDOW" are events sent by the render process to tell the main process to show/hide the AOT. I wanted to keep the existing logic for show/hide AOT.

Reply:
Thanks for the explanation